app/forms/collection_form.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Form for a Collection
+class CollectionForm < ApplicationForm
+  accepts_nested_attributes_for :related_links
+
+  def self.immutable_attributes
+    ['druid']
+  end
+
+  attribute :druid, :string
+  alias id druid
+
+  def persisted?
+    druid.present?
+  end
+
+  attribute :lock, :string
+
+  attribute :version, :integer, default: 1
+
+  attribute :title, :string
+  validates :title, presence: true
+
+  attribute :abstract, :string
```

Comment:
Collections have abstracts?

Reply:
They allow for it, at least in the descriptive, but perhaps we never use them. I'll remove.

Reply:
@justinlittman Yes, they appear to: https://argo.stanford.edu/view/druid:cm517jf6980

Reply:
Our beta collection: https://sdr.stanford.edu/collection_versions/241